forms/NumericField.php

```diff
@@ -2,13 +2,63 @@
 
 /**
  * Text input field with validation for numeric values. Supports validating
- * the numeric value as to the {@link i18n::get_locale()} value.
+ * the numeric value as to the {@link i18n::get_locale()} value, or an
+ * overridden locale specific to this field.
  * 
  * @package forms
  * @subpackage fields-formattedinput
  */
 class NumericField extends TextField {
 
+	/**
+	 * @var string
+	 */
+	protected $locale = null;
+
+	public function setValue($value, $data = array()) {
```

Comment:
PHPDoc for setValue? It's already specified on `FormField::setValue` and doesn't need to be redefined. It should inherit automatically.